### DISCUSSION TITLE
consider adding an explicit import of `is_nullable` to disambiguate when a table contains `is_nullable` column.

### DISCUSSION THREAD

**Discussion by forest1102** [diesel-rs/diesel#3403] [Q&A] [answered]
Hello, I am trying to update diesel 2.0, and I have a problem with a table that has an `is_nullable` column.  
Even though by adding `sql_name` attribute helps, I have to modify the schema file every time I changed the schema file. 
Suppose I made a migration file like below. 

```sql
create table db_definition_items
(
    id bigserial primary key,
    is_nullable boolean not null
);
```
and after `diesel database reset`, it will produce table schema below. 

```rs
diesel::table! {
    db_definition_items (id) {
        id -> Int8,
        is_nullable -> Bool,
    }
}
```

But this code has an error saying, 
```
`is_nullable` is ambiguous
ambiguous because of multiple glob imports of a name in the same module
consider adding an explicit import of `is_nullable` to disambiguate
consider adding an explicit import of `is_nullable` to disambiguate
```

How can I suppress this error?

## versions

```bash
$ rustc -V         
rustc 1.65.0 (897e37553 2022-11-02)

$ diesel --version
diesel 2.0.1
```

### Cargo.toml

```Cargo.toml
[package]
name = "serve"
version = "0.1.0"
edition = "2021"

# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html

[dependencies]
diesel = "2.0.2"
```

**Comment by weiznich** [comment, marked as answer]
Thanks for reporting. That's a bug, so I've opened a new issue for this in #3404. Will add more information there.

As a temporary workaround you can rename the column to something else in the `table!` macro call + use `#[sql_name = "is_nullable"]`.
